src/components/gauge/gauge.e2e.ts

```diff
@@ -0,0 +1,213 @@
+import { E2EElement, E2EPage, newE2EPage } from "@stencil/core/testing";
+describe("gx-gauge", () => {
+  let element: E2EElement;
+  let page: E2EPage;
+
+  beforeEach(async () => {
+    page = await newE2EPage();
+  });
+
+  it("should be drawn without attributes attrs", async () => {
+    await page.setContent("<gx-gauge></gx-gauge>");
+    await page.waitForChanges();
+    element = await page.find("gx-gauge");
+    expect(element).toBeTruthy();
+    expect(await element.find(".gaugeContainerLine")).toBeTruthy();
+  });
+
+  /////////////////// Type test ///////////////////////
+  it("should render with the line type", async () => {
+    await page.setContent("<gx-gauge type='line'></gx-gauge>");
+    await page.waitForChanges();
+    element = await page.find("gx-gauge");
+    await page.waitForChanges();
+    expect(await element.find(".gaugeContainerLine")).toBeTruthy();
+  });
+
+  it("should render with the circle type", async () => {
+    await page.setContent(`
+      <gx-gauge type="circle"></gx-gauge>
+    `);
+    await page.waitForChanges();
+    element = await page.find("gx-gauge");
+    await page.waitForChanges();
+    const svg = await element.find("svg");
+    const svgCircle = await element.find("svg circle");
+    await page.waitForChanges();
+    expect(svg).toBeTruthy();
+    expect(svgCircle).toBeTruthy();
+  });
+
+  /////////////////// Values test ///////////////////////
+  it("should automatically set the default minimum value", async () => {
+    await page.setContent(`
+      <gx-gauge show-value="true"></gx-gauge>
+    `);
+    await page.waitForChanges();
+    element = await page.find("gx-gauge");
+    const minValueDisplayer = await element.find("span.minValue");
+    expect(await element.getProperty("minValue")).toEqual(0);
+    expect(minValueDisplayer.innerHTML).toEqual("0<span></span>");
+  });
+
+  it("should set the passed minimum value", async () => {
+    const minValueTest = -100;
+    await page.setContent(`
+      <gx-gauge show-value="true" min-value="${minValueTest}"></gx-gauge>
+    `);
+    await page.waitForChanges();
+    element = await page.find("gx-gauge");
+    const minValueDisplayer = await element.find(
+      "div.minMaxDisplay span.minValue"
+    );
+    expect(await element.getProperty("minValue")).toEqual(minValueTest);
+    expect(minValueDisplayer.innerHTML).toEqual(`${minValueTest}<span></span>`);
+  });
```

Comment:
This test case seems very similar to the previous one. Maybe we can have only one that verifies both the initial default value and the value set in the `minValue` property.

Something like this:

```
  it("should set the passed minimum value", async () => {
    await page.setContent(`
      <gx-gauge show-value="true"></gx-gauge>
    `);
    await page.waitForChanges();
    element = await page.find("gx-gauge");
    const minValueDisplayer = await element.find("span.minValue");
    expect(minValueDisplayer.textContent).toEqual("0");
    
    const minValueTest = -100;
    await element.setProperty("minValue", minValueTest);
    await page.waitForChanges();
    expect(minValueDisplayer.textContent).toEqual(`${minValueTest}`);
  });
```

Reply:
🤔

Reply:
Here test two aspects. The first one checks if the default value of the control is assigned if no value is passed. The second one checks if the control takes and assigns the passed value. Do you suggest to merge both tests into only one?

Reply:
Yes, exactly. DRY also applies to tests 😉

Reply:
Check the commit 4e26143, there I merge both tests (checks if the default value of the control is assigned if no value is passed & checks if the control takes and assigns the passed value)